charts/hdfs-namenode-k8s/templates/namenode-statefulset.yaml

```diff
@@ -75,3 +121,13 @@ spec:
         - name: hdfs-name
           hostPath:
             path: {{ .Values.nameNodeHostPath }}
+        {{- if .Values.kerberosEnabled }}
+        - name: kerberos-config
+          configMap:
+            name: {{ .Values.kerberosConfigMap }}
+        - name: kerberos-keytabs
```

Comment:
Ditto. Rename to `hdfs-kerberos-keytabs`?

Reply:
Please see my response above.